ai_diffusion/comfy_client.py

```diff
@@ -340,9 +342,34 @@ async def clear_queue(self):
             except asyncio.QueueEmpty:
                 break
 
-        await self._post("queue", {"clear": True})
+        remote_ids = [await job.get_remote_id() for job in self._jobs]
+        await self._post("api/queue", {"delete": remote_ids})
+
         self._jobs.clear()
 
+    async def _poll_server_queue(self):
+        queue = await self._get("api/queue")
+        server_jobs = queue["queue_running"] + queue["queue_pending"]
+        # why are they unsorted to start with...?
+        server_jobs = sorted(server_jobs, key=itemgetter(0))
+        server_jobs = [entry[1] for entry in server_jobs]
+        if not (self._jobs or self._active):
```

Comment:
Not really sure about reporting general server state/activity when we're not waiting for a job. I think it's more likely to be confusing than helpful if you don't know how to inerpret the icon exactly.

Reply:
I'll drop it in the next version. I don't really like the whole iconography, honestly.

Reply:
dropped